mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -813,438 +840,538 @@ class vehicles {
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { {"ammo" }};
-				exclude[] = { {"airport" }};
+			class boat {
+				name = "Stab";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "naval", "stab" } };
+				exclude[] = { { "pavn" } };
 			};
-			class repair {
-				name = "Repair";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { {"repair" }};
-				exclude[] = { {"airport" }};
+			class RHIB {
+				name = "RHIB";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "naval" , "RHIB"} };
+				exclude[] = { { "pavn" } };
 			};
 		};
 	};
 
-	class utility_airport {
-		name = "Airport Utility";
-		RESPAWN_SHORT;
+	// green hornets /////////////////////////////////////////////////////////////////////
 
-		class categories {
-			class fuel {
-				name = "Fuel";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "fuel", "airport" } };
-				exclude[] = { { "pavn" } };
-			};
-			class ammo {
-				name = "Ammo";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "ammo", "airport" } };
-				exclude[] = { { "pavn" } };
-			};
-			class repair {
-				name = "Repair";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "repair", "airport" } };
-				exclude[] = { { "pavn" } };
-			};
-		};
+	class greenhornets_air_transport_all: parent_air_transport_all {
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_PUBLIC_HELOS;
 	};
 
-	class mp {
-		name = "MP Patrol";
-		RESPAWN_MEDIUM;
-		class categories {
-			class patrol {
-				name = "M151A1 Patrol";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "m151a1", "armed","mp" } };
-				exclude[] = { { "transport","usmc","antitank","army" } };
-			};
-			class transport {
-				name = "M151A1 Transport";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "m151a1", "transport","mp" } };
-				exclude[] = { { "armed","usmc","antitank","army" } };
-			};
-		};
-	};	
+	// acav + somewlus /////////////////////////////////////////////////////////////////////////
 
-	class acr {
-		name = "11th ACR Vehicles";
+	class acav_grnd_transport_heavy: parent_grnd_transport_heavy {
+		WRECK_LONG;
+		lockTeams[] = LOCKED_LIMITED_ARMOR;
+	};
+	class acav_grnd_firesupport_light: parent_grnd_firesupport_light {
+		WRECK_LONG;
+		lockTeams[] = LOCKED_LIMITED_ARMOR;
+	};
+	class acav_grnd_firesupport_heavy: parent_grnd_firesupport_heavy {
+		WRECK_LONG;
+		lockTeams[] = LOCKED_LIMITED_ARMOR;
+	};
+	class acav_statics_large: parent_statics_large {
 		RESPAWN_SHORT;
-		class categories {
-			class tanks {
-				name = "Tanks";
-				icon = VEHICLE_ICON_ARMOUR;
-				include[] = { { "tank","army" } };
-				exclude[] = { { "arvn","usmc","pavn"} };
-			};
-			class m113 {
-				name = "M113";
-				icon = VEHICLE_ICON_ARMOUR;
-				include[] = { { "m113","firesupport" } };
-				exclude[] = { { "Transport", "pavn" } };
-			};
-			class trucks {
-				name = "Trucks";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "truck","firesupport" } };
-				exclude[] = { { "pavn" } };
-			};
-		};
+		lockTeams[] = LOCKED_LIMITED_ARMOR;
 	};
```

Comment:
renamed to `blackhorse_grnd_firesupport_custom`, no template for it. 


can we change this to a template?